Bioc guidelines

enio23 opened the pull request:
Request to pull the new features to the main branch

enio23 committed:
Update bioc_guidelines

Update code writing as according to bioc guidelines

enio23 committed:
Update writeObjectiveFunction.R

fixing small typo on this function

enio23 committed:
Update writeObjectiveFunction.R

fixing small typo on this function

enio23 committed:
Update writeObjectiveFunction.R

fixing small typo on this function

enio23 committed:
Update invokeCPLEX.R

remove pesky indeces

enio23 committed:
Update cplex path

cplex path instead of solver path

enio23 committed:
Update invokeCPLEX.R

removing ptm object

enio23 committed:
Cleanup functionalities

adding Cleanup functionalities

enio23 committed:
Update ilpBinaryT1.R

controling the cplexPath parameter

enio23 committed:
Update CellNOptR-vignette.Rnw

update name of the authors and a description where to find the cplex solver

enio23 committed:
Update ilpBinaryT1.Rd

pointing that cplexPath is madatory on the documentation

enio23 committed:
Update CellNOptR-vignette.Rnw

fix author names not centered on the vignette

gabora commented:
CellNOpt uses camel case. 
eg. create_binaries.R  should be renamed to createBinaries.R

gabora commented:
make sure that `R CMD check ./` runs without errors and warnings.

gabora committed:
fixing check() and BiocCheck() errors and warnings

enio23 committed:
Update DESCRIPTION

Addressing warning: "Import grDevices, graphics, stats, utils in DESCRIPTION as well as NAMESPACE."

enio23 committed:
Update DESCRIPTION

Addressing warning: "Update R version dependency from 3.5.0 to 3.6."

enio23 committed:
Revert "Update DESCRIPTION"

This reverts commit 8b9102eca775f8c0569bd5804752e8511819ba82.

enio23 committed:
Revert "Update DESCRIPTION"

This reverts commit e6c267b9d14680d74958806f959dcd6fb88a65a2.

enio23 committed:
Update DESCRIPTION - R version

Addressing warning: "Update R version dependency from 3.5.0 to 3.6."

enio23 committed:
address warnings

address warnings

enio23 committed:
fix cross documentation

fix cross documentation
